docs/_ldio/ldio-adapters/index.md

```diff
@@ -8,6 +8,13 @@ nav_order: 4
 
 # Linked Data Orchestrator Adapters
 
-The LDI Core module contains the components maintained by the VSDS team in order to accommodate the onboarding of LDES onboarders.
+Adapters are be used in conjunction with the LDI Input. They will transform the provided content into and internal Linked Data model and sends it down the pipeline for further processing.
 
-Each component can be wrapped in a desired implementation framework (LDI-orchestrator, NiFi, ...) to be used.
+## Overview
+
+| Adapter             | Description                                                                              | Advantages                                                     | Disadvantages                                                               |
+|:--------------------|:-----------------------------------------------------------------------------------------|:---------------------------------------------------------------|:----------------------------------------------------------------------------|
+| _JSON to JSON-LD_   | Receives JSON messages and adds linked data context to transform the messages to JSON-LD | Readability of JSON-LD format Easy to set up (plug in context) | Only works with JSON as input Slower performance when deserializing model   |
+| _NGSIv2 to NGSI-LD_ | Converts NGSIv2 to an NGSI-LD model                                                      |                                                                | Only works with NGSIv2 as input Slower performance when deserializing model |
```

Comment:
Like mentioned before, I'd maybe put this in a separate addedum at the end by saying that this is meant for a specific use case?